### ISSUE TITLE
How to install by pod for swift3 ??

### ISSUE THREAD

**Issue by HirotoIchinose** [bahlo/SwiftGif#39]
I cannot install SwiftGif for swift3.

My Podfile is ..

```
pod 'SwiftGifOrigin', '~> 1.6'
```

But I get the following error.

```
Analyzing dependencies
[!] Unable to satisfy the following requirements:

- `SwiftGifOrigin (~> 1.6)` required by `Podfile`
- `SwiftGifOrigin (~> 1.6)` required by `Podfile`

None of your spec sources contain a spec satisfying the dependency: `SwiftGifOrigin (~> 1.6)`.

You have either:
 * out-of-date source repos which you can update with `pod repo update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

Note: as of CocoaPods 1.0, `pod repo update` does not happen on `pod install` by default.
```

I run `pod repo update`, but I get the same error again.

How can I use SwiftGif??

**Comment by LamourBt** [comment]
I have also run into this issue, the way i fixed it was to remove ~.1.6. Then do a `pod install` then a `pod update`

![screen shot 2016-09-29 at 5 42 20 am](https://cloud.githubusercontent.com/assets/14245117/18949042/84ad8f82-8607-11e6-92ce-12403816dda2.png)

**Comment by jvk75** [comment]
Swithcing to 1.5.3 is not a real solution as it is not Swift3. 1.5.3 cannot be build in Xcode8.
Only real solution is wait so that pod autoupdate picks in or the author issues force update.
1.6 is not visible in  cocopods

**Comment by bahlo** [comment]
Hey guys, sorry for the trouble. Will look into it asap 👍

**Comment by HarrisCarney** [comment]
@bahlo looks like this can be fixed by changing the folder "SwiftGifCommon" to "SwiftGifOrigin".

**Comment by matti777** [comment]
Hi, a Swift3 -compatible pod release coming out soonish?

**Comment by bahlo** [comment]
I'm so sorry guys, I'm drowning in work, will try and fix it this week. 💪

**Comment by Drenadan** [comment]
Hey, @bahlo should we wait for a fix in the near future?

**Comment by bahlo** [comment]
Closed via [`v1.6.1`](https://github.com/bahlo/SwiftGif/releases/tag/v1.6.1), thank you all for waiting.

BTW: If anyone is interested in becoming a maintainer of SwiftGif, please let me know - I barely find time as you may have noticed

**Comment by Drenadan** [comment]
@bahlo Thank You ! now it works.
